packages/cloud-cognitive/src/components/ComboButton/ComboButton.js

```diff
@@ -12,100 +12,154 @@ import {
+    ref
+  ) => {
+    const { current: instanceId } = useRef(uuidv4());
+    const [isOpen, setIsOpen] = useState(false);
+    const [primaryAction, ...restActions] = Children.toArray(children)
+      .filter(Boolean)
+      .map(({ props: { children, ...props } }) => {
+        return {
+          ...props,
+          children: (
+            <span className={`${blockClass}__action`} title={children}>
+              {children}
+            </span>
+          ),
+        };
+      });
+
+    return (
+      <div
+        {...rest}
+        className={cx(blockClass, className)}
+        data-floating-menu-container
+        ref={ref}
+      >
+        <Button
+          {...primaryAction}
+          size={size}
+          className={`${blockClass}__btn--${size}`}
+          disabled={disabled}
+        />
+
+        {restActions.length > 0 && (
+          <OverflowMenu
+            {...overflowMenu}
+            className={cx(
+              `${blockClass}__overflow-menu ${blockClass}__overflow-menu--${size}`
+            )}
+            disabled={disabled}
+            menuOptionsClass={`${blockClass}__overflow-menu__list`}
+            onClick={() => !isOpen && setIsOpen(true)}
+            onClose={() => setIsOpen(false)}
+            renderIcon={() =>
+              createElement(isOpen ? ChevronUp16 : ChevronDown16, {
+                className: cx(`${blockClass}__overflow-menu__icon`),
+              })
+            }
+            size={size}
+            flipped
+          >
+            {restActions.map(
+              ({ children, renderIcon: Icon, ...action }, index) => {
+                return (
+                  <OverflowMenuItem
+                    {...action}
+                    key={`${blockClass}--${instanceId}__overflow-menu__item__${index}`}
+                    className={cx(`${blockClass}__overflow-menu__item`)}
+                    itemText={
+                      <>
+                        {children}
 
+                        {Icon && (
+                          <span
+                            className={`${blockClass}__overflow-menu__item__icon`}
+                          >
+                            <Icon />
+                          </span>
+                        )}
+                      </>
+                    }
+                    size={size}
+                  />
+                );
+              }
+            )}
+          </OverflowMenu>
+        )}
+      </div>
+    );
+  }
+);
+
+ComboButton = pkg.checkComponentEnabled(ComboButton, componentName);
+ComboButton.displayName = componentName;
 ComboButton.propTypes = {
+  /** Provide an optional flag to disable entire ComboButton */
+  buttonDisabled: bool,
+
   /** Provide the contents of the `ComboButton` */
   children: node.isRequired,
 
   /** Provide an optional class to be applied to the containing node */
   className: string,
 
+  /** Provide an optional flag to disable the ComboButton or a selected ComboButtonItem */
+  disabled: bool,
```

Comment:
This does only disables the `ComboButton` (and the overflow menu) and not its children. Let’s update this for accuracy.